models/mail/src/index.ts

```diff
@@ -74,3 +72,24 @@ export function createModel (builder: Builder): void {
     component: mail.component.MailThread
   })
 }
+
+function createMailViewlet (builder: Builder): void {
+  builder.createDoc<Viewlet>(
+    view.class.Viewlet,
+    core.space.Model,
+    {
+      attachTo: mail.class.MailThread,
+      descriptor: view.viewlet.Table,
+      config: [
+        { key: 'createdBy', label: mail.string.From, displayProps: { fixed: 'left', key: 'app' } },
+        '',
+        { key: 'modifiedOn', label: mail.string.Date, displayProps: { key: 'modified', fixed: 'right' } }
```

Comment:
Why?

Reply:
If it only for label better override it using `declare` in model

Reply:
I've removed custom labels